src/Microsoft.Azure.NotificationHubs/NotificationHubClient.cs

```diff
@@ -2010,11 +2010,13 @@ private async Task DeleteRegistrationImplAsync(string registrationId, string eta
             using (var request = CreateHttpRequest(HttpMethod.Get, requestUri.Uri, out var trackingId))
             using (var response = await SendRequestAsync(request, trackingId, HttpStatusCode.OK, cancellationToken).ConfigureAwait(false))
             {
+                if (response.Content == null)
+                    return default;
```

Comment:
@JakubHolovsky the CI build for this PR is failing on our Windows build agent with the following message:

```
NotificationHubClient.cs(2014,28): error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater. [D:\a\1\s\src\Microsoft.Azure.NotificationHubs\Microsoft.Azure.NotificationHubs.csproj]
```

While the `default` keyword is handy when the generic type isn't constrained, in this case `TEntity` is constrained as `where TEntity : EntityDescription`. I think it's cleaner and more clear to return `null` and that would avoid the build error as well.

Can you make this change to your PR? I'll check to make sure the change fixes the CI build.

Thanks!

Reply:
Hello, sure thing, changed default to null. Should be all good now.